src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/Html/DefaultHtmlGenerator.cs

```diff
@@ -413,7 +413,14 @@ public virtual HtmlString GenerateAntiforgery([NotNull] ViewContext viewContext)
             var fullName = GetFullHtmlFieldName(viewContext, expression);
             if (string.IsNullOrEmpty(fullName))
             {
-                throw new ArgumentException(Resources.ArgumentCannotBeNullOrEmpty, nameof(expression));
+                throw new ArgumentException(
+                        Resources.FormatHtmlGenerator_FieldNameCannotBeNullOrEmpty(
+                            typeof(IHtmlHelper).FullName,
+                            nameof(IHtmlHelper.Editor),
+                            typeof(IHtmlHelper<>).FullName,
+                            nameof(IHtmlHelper<object>.EditorFor),
+                            "htmlFieldName"),
+                        nameof(expression));
```

Comment:
the spaces are for arguments to the Resources.FormatHtmlGenerator_FieldNameCannotBeNullOrEmpty method as it is a level deeper.